space.fips.Fips.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Matwey V. Kornilov <matwey.kornilov@gmail.com> -->
+<component type="desktop-application">
+<id>space.fips.Fips</id>
+<metadata_license>FSFAP</metadata_license>
+<project_license>LGPL-3.0+</project_license>
+<name>FIPS</name>
+<summary>OpenGL-based FITS image viewer</summary>
+<description>
+	<p>
+FIPS is a cross-platform FITS viewer with responsive user interface. Unlike
+other FITS viewers FIPS uses GPU hardware via OpenGL to provide usual
+functionality such as zooming, panning and level adjustments. OpenGL 2.1 and
+later is supported.
+	</p>
+	<p>
+FIPS supports all 2D image formats (excluding floating point for OpenGL 2.1).
+FITS image extension has basic limited support.
+	</p>
+</description>
+<screenshots>
+	<screenshot type="default">https://raw.githubusercontent.com/matwey/flathub/fips/screenshots/1.png</screenshot>
+</screenshots>
+<launchable type="desktop-id">space.fips.Fips.desktop</launchable>
+<url type="homepage">https://fips.space</url>
```

Comment:
You could add some other urls like issues, translations (if there's any url for that), donation! you could also use the update_contact, developper_name and other tags to make sure that your app look quite good on any linux software center :+1: